model/search.go

```diff
@@ -15,6 +15,7 @@ type Search struct {
 	ErrorMessage string           `json:"error_message,omitempty"`
 	Filter       []string         `json:"filter,omitempty"`
 	Filters      []Filter         `json:"filters"`
+	TopicFilters []TopicFilter    `json:"topic_filters"`
```

Comment:
Can just use Filter struct

```suggestion
	TopicFilters []Filter    `json:"topic_filters"`
```